ecosystem.config.yml

```diff
@@ -0,0 +1,7 @@
+apps:
+  - script: ./src/index.js
+    name: 'memebot-api'
+    env:
+      NODE_ENV: production
```

Comment:
Need to double check code assumptions on NODE_ENV.

Reply:
Good call, I should probably try running that locally first.